### DISCUSSION TITLE
Bun v1.0.21

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#7944] [Announcements]
To install Bun v1.0.21:
```bash
curl https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

To upgrade to Bun v1.0.21:
```bash
bun upgrade
```

**[Read Bun v1.0.21's release notes on Bun's blog](https://bun.sh/blog/bun-v1.0.21)**

## Thanks to 20 contributors who made this release possible!

- [@aarvinr](https://github.com/aarvinr)
- [@cirospaciari](https://github.com/cirospaciari)
- [@coratgerl](https://github.com/coratgerl)
- [@Hanaasagi](https://github.com/Hanaasagi)
- [@huseeiin](https://github.com/huseeiin)
- [@Jarred-Sumner](https://github.com/Jarred-Sumner)
- [@jeroenpg](https://github.com/jeroenpg)
- [@johnpyp](https://github.com/johnpyp)
- [@kaioduarte](https://github.com/kaioduarte)
- [@knightspore](https://github.com/knightspore)
- [@L422Y](https://github.com/L422Y)
- [@malthe](https://github.com/malthe)
- [@nektro](https://github.com/nektro)
- [@o-az](https://github.com/o-az)
- [@o2sevruk](https://github.com/o2sevruk)
- [@otgerrogla](https://github.com/otgerrogla)
- [@RiskyMH](https://github.com/RiskyMH)
- [@rtxanson](https://github.com/rtxanson)
- [@thunfisch987](https://github.com/thunfisch987)
- [@xlc](https://github.com/xlc)

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v1.0.21'>Bun v1.0.21</a>.</em>

**Comment by ddevienne** [comment]
Just a quick note to thank you for the great release notes you always link to (and write I suspect) @Jarred-Sumner.
These are probably the best ones I've ever read. Even to a non-JS/TS dev like me more into statically-typed and natively-compiled langs.
And sorry for the noise here, but I didn't see how else to tell you.

**Comment by Hugos68** [comment]
There is a typo in the blogpost:
```js
const foo = await fetch("https://api.github.com/repos/elyisajs/elysia/issues");
console.table(await issues.json(), ["title", "status"]);
```
response is called `foo` but the json is pulled from `issues` var
